runtime/src/lib.rs

```diff
@@ -268,7 +268,7 @@ impl pallet_utility::Config for Runtime {
 }
 
 // Existential deposit.
-pub const EXISTENTIAL_DEPOSIT: u64 = 500;
+pub const EXISTENTIAL_DEPOSIT: u64 = 2_000_000;
```

Comment:
Do we know the implications of this change? If ED is increased, would this affect wallets that have staked tao in their hotkey but 0 tao in their coldkey? It would be problematic for users if all of their tao is staked and then suddenly their cold wallet is destroyed, orphaning their hot wallet. 

I suppose it's always possible to regain the wallet if you have the mnemonic, but it can just be annoying.